markdown/design-to-code-workshop/design-to-code-workshop.md

```diff
@@ -0,0 +1,559 @@
+author: Marc DiPasquale
+id: design-to-code-workshop
+summary: This codelab walks you through how to design an EDA using the Solace PubSub+ Event Portal, export the AsyncAPI documents, generate code using the AsyncAPI generator and see your events flow across your apps!
+tags: AsyncAPI, Spring, Java, Portal
```

Comment:
Need to add "Solace" first or have an AsyncAPI style created.

Reply:
We _could_ add a workshop specific style if we wanted to but yea Solace is good

Reply:
changing to Solace for now, but I do think we should add an AsyncAPI category since we'll eventually have hopefully codelabs for using the different generator templates

Reply:
Opened this to capture: https://github.com/SolaceDev/solace-dev-codelabs-site/issues/54